<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>AddVehicle</class>
 <widget class="QDialog" name="AddVehicle">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>583</width>
    <height>423</height>
   </rect>
  </property>
  <property name="font">
   <font>
    <family>Avenir Next</family>
   </font>
  </property>
  <property name="windowTitle">
   <string>Kup samochód</string>
  </property>
  <widget class="QGroupBox" name="formGroupBox">
   <property name="geometry">
    <rect>
     <x>9</x>
     <y>10</y>
     <width>561</width>
     <height>261</height>
    </rect>
   </property>
   <layout class="QGridLayout" name="gridLayout">
    <item row="0" column="0">
     <widget class="QLabel" name="Model">
      <property name="text">
       <string>Model:</string>
      </property>
     </widget>
    </item>
    <item row="2" column="0">
     <widget class="QLabel" name="Cena">
      <property name="text">
       <string>Cena:</string>
      </property>
     </widget>
    </item>
    <item row="5" column="1">
     <widget class="QComboBox" name="Paliw_wybierz"/>
    </item>
    <item row="4" column="1">
     <widget class="QComboBox" name="Stan_wybierz"/>
    </item>
    <item row="5" column="0">
     <widget class="QLabel" name="Paliwo">
      <property name="text">
       <string>Silnik:</string>
      </property>
     </widget>
    </item>
    <item row="0" column="1">
     <widget class="QComboBox" name="Model_wybierz"/>
    </item>
    <item row="4" column="0">
     <widget class="QLabel" name="Stan">
      <property name="text">
       <string>Stan:</string>
      </property>
     </widget>
    </item>
    <item row="3" column="1">
     <widget class="QTextEdit" name="ID_pisz">
      <property name="maximumSize">
       <size>
        <width>500</width>
        <height>30</height>
       </size>
      </property>
     </widget>
    </item>
    <item row="3" column="0">
     <widget class="QLabel" name="ID">
      <property name="text">
       <string>ID:</string>
      </property>
     </widget>
    </item>
    <item row="2" column="1">
     <widget class="QTextEdit" name="Cena_pisz">
      <property name="maximumSize">
       <size>
        <width>16777215</width>
        <height>30</height>
       </size>
      </property>
     </widget>
    </item>
    <item row="6" column="1">
     <widget class="QComboBox" name="Skrzynia_biegow_wybierz"/>
    </item>
    <item row="6" column="0">
     <widget class="QLabel" name="Skrzyni_biegow">
      <property name="text">
       <string>Skrzynia biegów:</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QGroupBox" name="horizontalGroupBox">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>330</y>
     <width>501</width>
     <height>71</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout">
    <item>
     <widget class="QPushButton" name="Akceptuj">
      <property name="text">
       <string>Akceptuj</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="Anuluj">
      <property name="text">
       <string>Anuluj</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
